src/Symfony/Component/Ldap/Adapter/ExtLdap/Collection.php

```diff
@@ -51,9 +51,9 @@ public function count()
         $searches = $this->search->getResources();
         $count = 0;
         foreach ($searches as $search) {
-            $searchCount = ldap_count_entries($con, $search);
+            $searchCount = @ldap_count_entries($con, $search);
             if (false === $searchCount) {
-                throw new LdapException(sprintf('Error while retrieving entry count: %s.', ldap_error($con)));
+                throw LdapException::create('Error while retrieving entry count: [{errorCode}] {errorMsg}.', ldap_errno($con));
```

Comment:
We are not doing this in Symfony, can you please revert the `::create()` method?